src/Sentry/Experimental/System.Diagnostics.CodeAnalysis.ExperimentalAttribute.cs

```diff
@@ -0,0 +1,29 @@
+#if !NET8_0_OR_GREATER
+// ReSharper disable CheckNamespace
+// ReSharper disable ConvertToPrimaryConstructor
+namespace System.Diagnostics.CodeAnalysis;
+
+[AttributeUsage(AttributeTargets.Assembly |
+                AttributeTargets.Module |
+                AttributeTargets.Class |
+                AttributeTargets.Struct |
+                AttributeTargets.Enum |
+                AttributeTargets.Constructor |
+                AttributeTargets.Method |
+                AttributeTargets.Property |
+                AttributeTargets.Field |
+                AttributeTargets.Event |
+                AttributeTargets.Interface |
+                AttributeTargets.Delegate, Inherited = false)]
+internal sealed class ExperimentalAttribute : Attribute
```

Comment:
@SimonCropp has a shimming library that might have this, no?

Reply:
It is indeed!
I only checked the `README` where I didn't find it, but since `v1.32.0` it is included (I'll PR `Polyfill`'s docs later).

Now I got two questions:
- Should we update to `v1.32.0`, or the latest version of `Polyfill` (`v7.27.0`).
- Depending on the version we chose: should we update this in a separate PR?
  - e.g. `v1.34.0` adds the `System.Runtime.CompilerServices.DefaultInterpolatedStringHandler` which the compiler then chooses for _interpolated strings_
    - this should have a positive effect on performance when using _interpolated strings_ in our codebase targeting TFMs less than `net6.0`

Reply:
see #4167

Reply:
done